jormungandr-lib/src/time.rs

```diff
@@ -273,15 +273,13 @@ impl From<u64> for SystemTime {
     }
 }
 
-impl From<u64> for SecondsSinceUnixEpoch {
-    fn from(seconds_since_epoch: u64) -> Self {
+impl SecondsSinceUnixEpoch {
+    pub fn from_secs_since_epoch(seconds_since_epoch: u64) -> Self {
```

Comment:
This looks a bit tautological, but OK.

Reply:
no, `from_secs` would be better. It will already be `SecondsSinceUnixEpoc::from_secs`. No need to specify again `from_secs_since_epoch`.